wzdx/experimental_combination/attenuator.py

```diff
@@ -0,0 +1,285 @@
+import json
+from ..tools import (
+    cdot_geospatial_api,
+    geospatial_tools,
+    combination,
+    date_tools,
+    wzdx_translator,
+)
+import logging
+from datetime import datetime, timedelta
+
+ATTENUATOR_TIME_AHEAD_SECONDS = 30 * 60
+ISO_8601_FORMAT_STRING = "%Y-%m-%dT%H:%M:%SZ"
+
+
+def main(outputPath="./tests/data/output/wzdx_attenuator_combined.json"):
+    with open(
+        "./wzdx/sample_files/raw/geotab_avl/attenuator_combination_geotab.json"
+    ) as f:
+        geotab_avl = [json.loads(f.read())]
+    with open(
+        "./wzdx/sample_files/enhanced/attenuator/attenuator_combination_wzdx.json"
+    ) as f:
+        wzdx = json.loads(f.read())
+        wzdx[0]["features"][0]["properties"]["start_date"] = (
+            date_tools.get_iso_string_from_datetime(datetime.now() - timedelta(days=1))
```

Comment:
The WZDx feed here already has a start date right? Are we ok just modifying it to "yesterday" if it already exists? What if it exists in the future?